english/marshmallow.md

```diff
@@ -919,4 +919,6 @@ Toasted [marshmallow](toasted/smores_pie.md)
 While having toasted marshmellows and watching the warriors beat the grizzlies, you realise they set a new record.
 [The new record is!](new_warriors_record/new_warriors_record.md)
 
-Of Course you must reach the end of your story, but remember [this](interstellar_quote/light.md).
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.